packages/labs/virtualizer/package.json

```diff
@@ -9,34 +9,52 @@
   "type": "module",
   "exports": {
     ".": {
-      "default": "./lit-virtualizer.js"
+      "default": "./lit-virtualizer.js",
+      "types": "./lit-virtualizer.d.ts"
     },
     "./events.js": {
-      "default": "./events.js"
+      "default": "./events.js",
+      "types": "./events.d.ts"
     },
     "./layouts/flexWrap.js": {
-      "default": "./layouts/flexWrap.js"
+      "default": "./layouts/flexWrap.js",
+      "types": "./layouts/flexWrap.d.ts"
     },
     "./layouts/flow.js": {
-      "default": "./layouts/flow.js"
+      "default": "./layouts/flow.js",
+      "types": "./layouts/flow.d.ts"
     },
     "./layouts/grid.js": {
-      "default": "./layouts/grid.js"
+      "default": "./layouts/grid.js",
+      "types": "./layouts/grid.d.ts"
     },
     "./layouts/masonry.js": {
-      "default": "./layouts/masonry.js"
+      "default": "./layouts/masonry.js",
+      "types": "./layouts/masonry.d.ts"
     },
     "./LitVirtualizer.js": {
-      "default": "./LitVirtualizer.js"
+      "default": "./LitVirtualizer.js",
+      "types": "./LitVirtualizer.d.ts"
     },
     "./polyfillLoaders/ResizeObserver.js": {
-      "default": "./polyfillLoaders/ResizeObserver.js"
+      "default": "./polyfillLoaders/ResizeObserver.js",
+      "types": "./polyfillLoaders/ResizeObserver.d.ts"
     },
     "./polyfills/resize-observer-polyfill/ResizeObserver.js": {
-      "default": "./polyfillLoaders/ResizeObserver.js"
+      "default": "./polyfillLoaders/ResizeObserver.js",
+      "types": "./polyfillLoaders/ResizeObserver.d.ts"
```

Comment:
is it intentional that these lines don't actually map to the `./polyfills/resize-observer-polyfill/RseizeObserver.js` path? if not, it probably should be fixed. that particular file also doesn't seem to have a `d.ts` file in the same directory. looks like the original source is `js` with a separate `d.ts` file. maybe we need to copy over the `d.ts` file too. sorry this could be done in a separate PR than this one. it just caught my eye.

Reply:
The `/polyfillLoaders/ResizeObserver.js` is actually the "loader" for the polyfill, not the actual polyfill, if that makes sense?

Reply:
hard to see just from the selection but the path and the target are different.
https://github.com/lit/lit/blob/5d68be35c192e8c4109911eec727fbb598557f72/packages/labs/virtualizer/package.json#L32-L37